nifi-nar-bundles/nifi-standard-services/nifi-lookup-services-bundle/nifi-lookup-services/src/main/java/org/apache/nifi/lookup/RestLookupService.java

```diff
@@ -172,6 +172,17 @@ public class RestLookupService extends AbstractControllerService implements Reco
         .addValidator(StandardValidators.TIME_PERIOD_VALIDATOR)
         .build();
 
+
+    public static final PropertyDescriptor PROP_RESPONSE_CODE_HANDLING = new PropertyDescriptor.Builder()
```

Comment:
Recommend removing the `PROP_` prefix:
```suggestion
    public static final PropertyDescriptor RESPONSE_HANDLING_STRATEGY = new PropertyDescriptor.Builder()
```